keen/explorer
# Fixes infinite loop when rendering chart

## What does this PR do? How does it affect users?

It fixes a bug where Explorer would get into an infinite loop when rendering the chart. This was because we were using state to keep track of the lastDataTimestamp, which caused re-renders. Instead, we're just using a component property.

## How should this be tested?

Try loading different saved queries one after the other and ensure they do not cause an infinite loop inside of `showVisualization` inside of the `KeenViz` component.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:
